tests/unit/account_id_test.py

```
@@ -73,6 +81,19 @@ def test_initialization_with_alias_key(alias_key):
     assert account_id.alias_key == alias_key
     assert account_id.checksum is None
+    assert account_id.evm_address is None
+
+def test_initialization_with_evm_address(evm_address):
+    """Test AccountId initialization with alias key."""
+    account_id = AccountId(shard=0, realm=0, num=0, evm_address=evm_address)
+
+    assert account_id.shard == 0
+    assert account_id.realm == 0
+    assert account_id.num == 0
+    assert account_id.alias_key == None
```

Ruff (0.14.10)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)